src/main/java/com/telefonica/iot/cygnus/sinks/OrionPostgreSQLSink.java

```diff
@@ -141,7 +141,7 @@ public void configure(Context context) {
         postgresqlPassword = context.getString("postgresql_password", "unknown");
```

Comment:
I've seen this "unknown" default password... I guess all the default passwrods are empty in the templates, right?